pkg/vpcmodel/semanticDiffSubnets.go

```diff
@@ -126,25 +126,23 @@ func (subnetConfConnectivity *SubnetConfigConnectivity) subtract(other *SubnetCo
 			if err2 != nil {
 				return nil, err2
 			}
+			connDiff := &connectionDiff{conns, nil, MissingConnection}
 			if srcInOther != nil && dstInOther != nil {
 				if otherSrc, ok := other.subnetConnectivity[srcInOther]; ok {
-					if otherSrcDst, ok := otherSrc[dstInOther]; ok {
+					if otherConn, ok := otherSrc[dstInOther]; ok {
 						// ToDo: https://github.com/np-guard/vpc-network-config-analyzer/issues/199
-						subtractConn := conns.Subtract(otherSrcDst)
+						subtractConn := conns.Subtract(otherConn)
 						if subtractConn.IsEmpty() {
```

Comment:
```suggestion
						if conns.Equals(otherConn) {
```

Reply:
No, this change is no good.
The reason is the discussion in the next item - the operand I calculate is not symmetric, as is`subtractConn.IsEmpty()`; however,  `conns.Equals(otherConn` is symmetric.

Reply:
done and enhanced to avoid dup lines in final presentation